server/models/meshery_application.go

```diff
@@ -15,17 +15,17 @@ type ApplicationTypeResponse struct {
 	SupportedExtensions []string        `json:"supported_extensions"`
 }
 
-func GetApplicationTypes() (r []DesignTypeResponse) {
-	r = append(r, DesignTypeResponse{
-		Type:                HelmChart,
+func GetApplicationTypes() (r []ApplicationTypeResponse) {
+	r = append(r, ApplicationTypeResponse{
+		Type:                ApplicationType(HelmChart),
```

Comment:
@theBeginner86, one of the possible solution is to make it return []ApplicationTypeResponse and explicity converting the DesignType to ApplicationType.